unittests/test_modules.py

```diff
@@ -401,3 +403,55 @@ def test_mapping_deep_cycle(self):
         }
         self.assertRaisesRegex(EnvironError, 'm0->m1->m3->m4->m2->m1',
                                self.modules_system.load_module, 'm0')
+
+    def test_mapping_from_file_simple(self):
+        self.fp.write('m1:m2  m3   m3\n'
+                      'm2 : m4 \n'
+                      ' #m5: m6\n'
+                      '\n'
+                      'm5: m6 # Inline comment')
```

Comment:
Having these two lines with `m5: m6`, you cannot tell which one was added. Please, use different names, like `m7: m8`.